src/app/services/ths-stateflow.service.ts

```diff
@@ -13,16 +13,15 @@ export class ThsStateflowService {
   // the answer to the last question, the patient may be asked to fill out a short text box
   // with some info so it wil change the state based on that
   public moveStateForward(state: number, choice: string): number {
+    console.log("state: " + state);
```

Comment:
Same as above: ' should be used for strings instead of quotation marks.

Reply:
Yes, I'll need to get in the habit of using ' instead of "